provider/google.go

```diff
@@ -182,7 +182,10 @@ func (p *googleProvider) DeleteZone(name string) error {
 func (p *googleProvider) Records(zone string) (endpoints []*endpoint.Endpoint, _ error) {
 	f := func(resp *dns.ResourceRecordSetsListResponse) error {
 		for _, r := range resp.Rrsets {
-			if r.Type != "A" {
+			switch r.Type {
+			case "A", "CNAME":
```

Comment:
for the type filtering, we will have to remove it once ownership concept will be introduced